PacketDefinitions420/PacketNotifier.cs

```diff
@@ -292,10 +292,26 @@ public void NotifyAvatarInfo(int userId, ClientInfo client)
             _packetHandlerManager.SendPacket(userId, avatar.GetBytes(), Channel.CHL_S2C);
         }
 
-        public void NotifyBuyItem(int userId, IChampion champion, IItem itemInstance)
+        public void NotifyBuyItem(int userId, IObjAiBase gameObject, IItem itemInstance)
         {
-            var buyItem = new BuyItemResponse(champion, itemInstance);
-            _packetHandlerManager.SendPacket(userId, buyItem, Channel.CHL_S2C);
+            ItemData itemData = new ItemData
+            {
+                ItemID = (uint)itemInstance.ItemData.ItemId,
+                Slot = gameObject.Inventory.GetItemSlot(itemInstance),
+                ItemsInSlot = itemInstance.StackCount,
+                SpellCharges = 0
+            };
+
+            var buyItemPacket = new BuyItemAns
+            {
+                Item = itemData,
+                SenderNetID = gameObject.NetId
+            };
+
+            // todo find out what bitfield does, currently unknown
+            //buyItemPacket.Bitfield = null;
```

Comment:
Remove commented code. and change `todo` -> `TODO: `

(See here: https://docs.microsoft.com/en-us/visualstudio/ide/using-the-task-list?view=vs-2019)